docs/src/udp.rst

```diff
@@ -47,6 +47,13 @@ Data types
              * must not be freed by the recv_cb callback.
              */
             UV_UDP_MMSG_CHUNK = 8,
+            /*
+             * Indicates if IP_RECVERR/IPV6_RECVERR will be set when binding the handle.
+             * This sets IP_RECVERR for IPv4 and IPV6_RECVERR for IPv6 UDP sockets on
+             * Linux. This stops the Linux kernel from suppressing some ICMP error messages
+             * and enables full ICMP error reporting for faster failover.
+             */
+            UV_UDP_LINUX_RECVERR = 16,
```

Comment:
I understand it's implied but may make clear that it's a noop in other platforms.

Reply:
Ok, added a short note.